lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/form/Auto.js

```diff
@@ -127,7 +127,7 @@ qx.Class.define("callbackery.ui.form.Auto", {
             if (s.set) {
                 if (s.widget == 'date') {
                     if (s.set.value) {
-                        var dateValue = s.set.value;
+                        var dateValue = s.set.value * 1000; // incoming epoch seconds
                         delete s.set.value;
```

Comment:
didn't you patch the Date setter elsewhere differently ? using a Date object created with the json string as input ? I guess it would be cool if this was consistent.